src/Components/Search/state.ts

```diff
@@ -27,19 +27,28 @@ export class SearchBarState extends Container<StateContainer> {
   reset() {
     this.setState({
       hasEnteredPreviews: false,
-      selectedPreviewIndex: 0,
+      selectedPreviewIndex: null,
       previewItems: [],
     })
   }
 
+  enterPreviewWithoutSelection() {
```

Comment:
When entering w/ the mouse, we can just always set this to true (it's already true if you're in there from the keyboard).